core/templates/core/user_accounts/password_change/password_change_done.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'core/base.html' %}
+{% load i18n static %} <!-- Serves translation -->
+
+{% block title %}
+  Squire - Password Change Done
+{% endblock title %}
+
+{% block content_title %}<h1>{{ title }}</h1>{% endblock %}
+{% block content %}
+<p>{% trans 'Your password was changed.' %}</p>
```

Comment:
This is the only translation in the whole template, shouldn't everything be translated?

Reply:
See the comment at https://github.com/EricTRL/Squire/pull/25#discussion_r326639283